--- setup.py.old	2009-03-19 17:22:06.000000000 -0400
+++ setup.py	2009-03-19 17:29:08.000000000 -0400
@@ -34,6 +34,7 @@
     use_numpy = False
     sys.argv.remove("--numarray")
 
+lib = "lib"
 if use_numeric:
     extra_compile_args.append("-DNUMERIC=1")
 elif use_numarray:
@@ -44,8 +45,10 @@
         arrayobject_h_include = [os.path.join(sys.prefix,
                                  "Lib/site-packages/numpy/core/include")]
     else:
+        if sys.arch == 'x86_64':
+            lib = "lib64"
         arrayobject_h_include = [os.path.join(sys.prefix,
-                                "lib/python%s.%s/site-packages/numpy/core/include"
+                                lib + "/python%s.%s/site-packages/numpy/core/include"
                                 % sys.version_info [:2])]
 
 math_libraries = []
@@ -77,8 +80,8 @@
         pass
 if netcdf_prefix is None:
     for netcdf_prefix in ['/usr/local', '/usr', '/sw']:
-        netcdf_include = os.path.join(netcdf_prefix, 'include')
-        netcdf_lib = os.path.join(netcdf_prefix, 'lib')
+        netcdf_include = os.path.join(netcdf_prefix, 'include' )
+        netcdf_lib = os.path.join(netcdf_prefix, lib)
         if os.path.exists(os.path.join(netcdf_include, 'netcdf.h')):
             break
     else:

