--- setup.py.orig	2013-07-04 10:20:49.378991841 +0200
+++ setup.py	2013-07-04 10:28:03.242471793 +0200
@@ -34,6 +34,7 @@ if "--numarray" in sys.argv:
     use_numpy = False
     sys.argv.remove("--numarray")
 
+lib = "lib"
 if use_numeric:
     extra_compile_args.append("-DNUMERIC=1")
 elif use_numarray:
@@ -46,6 +47,8 @@ else :
 math_libraries = []
 if sys.platform != 'win32':
     math_libraries.append('m')
+else :
+    math_libraries.append('m')
 
 #
 # Locate netCDF library
@@ -72,8 +75,8 @@ if netcdf_prefix is None:
         pass
 if netcdf_prefix is None:
     for netcdf_prefix in ['/usr/local', '/usr', '/sw']:
-        netcdf_include = os.path.join(netcdf_prefix, 'include')
-        netcdf_lib = os.path.join(netcdf_prefix, 'lib')
+        netcdf_include = os.path.join(netcdf_prefix, 'include' )
+        netcdf_lib = os.path.join(netcdf_prefix, lib)
         if os.path.exists(os.path.join(netcdf_include, 'netcdf.h')):
             break
     else:
